src/fcntl.rs

```diff
@@ -9,101 +9,26 @@ use sys::uio::IoVec;  // For vmsplice
 use libc;
```

Comment:
delete this import and the `#[cfg]` attribute, and change line 2 to `use libc::{self, c_int, c_uint}`. This is one reason for the Apple build failing: `libc` the module was only imported on Linux.